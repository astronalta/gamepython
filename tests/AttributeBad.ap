
BadAttribute < Object {


    bad_attribute String

    @init() {
        bad_attribute = 1 # Should cause a type error
        bad_attribute Any # Shadows attribute (not allowed for now)
    }
}
