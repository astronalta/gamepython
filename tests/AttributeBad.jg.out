AttributeBad.jg:8:25: Expression does not conform to type 'String' in assignment of 'bad_attribute'
AttributeBad.jg:9:9: Local variable declaration shadows attribute
